src/test/java/org/cloudfoundry/promregator/cfaccessor/CFAccessorCacheCaffeineTimeoutTest.java

```diff
@@ -31,15 +36,29 @@
 	
 	@Autowired
 	private CFAccessorCacheCaffeine subject;
-	
-	@Before
-	public void invalidateCaches() {
+
+	@BeforeEach
+	public void setup() {
+		Mockito.reset(this.parentMock);
```

Comment:
Just a stupid idea: The cache stuff below has nothing to do with this resetting/configuring of the mock. Hence, what about the idea of having two separate `@BeforeEach` methods for that to properly separate the semantics?

Reply:
I'm fine with that. I renamed the method to `setup` since it was mixing concerns. I'll actually be switching to `@MockBean` in an upcoming PR so the reset will probably be removed at that time.

Reply:
Done